modules/hitlnext/src/views/client.ts

```diff
@@ -101,9 +101,13 @@ export const makeClient = (bp: { axios: AxiosInstance }): HitlClient => {
         .then(res => res.data)
         .then(data => castHandoff(data)),
     deleteMessagesInChannelWeb: async (id, userId) =>
-      bp.axios.post(`/conversations/${userId}/${id}/messages/delete`, null, {
-        baseURL: bp.axios.defaults.baseURL.concat('/mod/channel-web')
-      }),
+      bp.axios.post(
+        '/conversations/messages/delete',
+        { conversationId: id, userId, webSessionId: (window as any).__BP_VISITOR_SOCKET_ID },
```

Comment:
@allardy This does not work since I simply fetch the VISITOR_SOCKET_ID and use it for both the agent and the user! Any idea how I can fetch the user's socket id?

Reply:
That is simply not doable as of now :/ 

And I don't see how we can expose that to the hitlnext module properly ... We might have to duplicate the channel web code for now.

Or we could add another secured endpoint in channel web to delete user messages

Reply:
fixed by adding a temp secured route, see my commit